hr_timesheet_sheet/models/hr_timesheet_sheet.py

```diff
@@ -807,6 +807,76 @@ def _track_subtype(self, init_values):
                 return self.env.ref("hr_timesheet_sheet.mt_timesheet_approved")
         return super()._track_subtype(init_values)
 
+    # ------------------------------------------------
+    # to prevent sending emails to the reviewer
+    # ------------------------------------------------
+
+    def get_reviewers(self, company, record_id):
```

Comment:
Why do you need this parameters?
There is [company_id](https://github.com/OCA/timesheet/blob/3bee29030c5f47d3977ac2c58f433bcea3c555f1/hr_timesheet_sheet/models/hr_timesheet_sheet.py#L126C5-L126C15) field which you can use. 
And what is the problem with current recordset that forces you to pass record_id explicitly?